docs/Makefile

```diff
@@ -1,7 +1,7 @@
-PREFIX := /usr/local
-DATADIR := ${PREFIX}/share
-MANDIR := $(DATADIR)/man
-GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')
+PREFIX ?= /usr/local
+DATADIR ?= ${PREFIX}/share
+MANDIR ?= $(DATADIR)/man
+MANDOWN ?= $(shell command -v mandown || echo '$(PATH)/mandown')
```

Comment:
How/Where is this installed?

Also appending like to $PATH is wrong, $PATH contains more than one path usually.  `command` already looks up $PATH so why try to use it anyway

Reply:
mandown will be installed in the rust ci image. I'll remove that PATH part.

Reply:
@Luap99 done. PTAL